demos/src/demo.mustache

```diff
@@ -1,105 +1,89 @@
-						<button class="o-subs-card__select-button">Select</button>
-					</div>
-				</div>
-				<div class="o-subs-card__copy-pitch">Access to FT.com on your desktop, mobile and tablet</div>
-				<div class="o-subs-card__expander">
-					<button class='o-subs-card__read-more'>Read more</button>
-					<div class="o-subs-card__copy-details">
-						<div class="o-subs-card__copy-preamble">Standard Digital benefits, plus</div>
-						<ul class="o-subs-card__copy-benefits">
-							<li>Access to FT's award-winning news on desktop, mobile and tablet</li>
-							<li>Personalised email briefings by industry, journalist or sector</li>
-						</ul>
-					</div>
-				</div>
+	</div>
+	<div class="o-subs-card" data-o-component="o-subs-card">
+		<div class="o-subs-card__top">
+			<div class="o-subs-card__copy-title">Standard Digital</div>
+			<div class="o-subs-card__copy-pitch">Be informed with the essential news and opinion</div>
+			<div class="o-subs-card__charge">
+				<button class="o-subs-card__select-button">Select</button>
+				<div class="o-subs-card__charge__value">£XX.XX per week</div>
+			</div>
+		</div>
+		<div class="o-subs-card__expander">
+			<div class="o-subs-card__copy-details">
+				<ul class="o-subs-card__copy-benefits">
+					<li>Access to FT's award-winning news on desktop, mobile and tablet</li>
+					<li>Personalised email briefings by industry, journalist or sector</li>
+					<li>Portfolio tools to help manage your investments</li>
+					<li>FastFT - market-moving news and views, 24 hours a day</li>
+					<li>Brexit Briefing - your essential guide to the impact of the UK-EU split</li>
+				</ul>
+			</div>
+			<div class='o-subs-card__read-more'>Read more</div>
+		</div>
+	</div>
+	<div class="o-subs-card o-subs-card--discount" data-o-component="o-subs-card">
+		<div class="o-subs-card__top">
+			<div class="o-subs-card__copy">
+				<div class="o-subs-card__copy-offer">Limited Time Offer</div>
+				<div class="o-subs-card__copy-title">Standard Digital</div>
+				<div class="o-subs-card__copy-pitch">Be informed with the essential news and opinion</div>
+			</div>
+			<div class="o-subs-card__charge">
+				<button class="o-subs-card__select-button">Select</button>
+				<div class="o-subs-card__charge__value">£XX.XX per week</div>
+			</div>
+		</div>
+		<div class="o-subs-card__expander">
+			<div class="o-subs-card__copy-details">
+				<ul class="o-subs-card__copy-benefits">
+					<li>Access to FT's award-winning news on desktop, mobile and tablet</li>
+					<li>Personalised email briefings by industry, journalist or sector</li>
+					<li>FastFT - market-moving news and views, 24 hours a day</li>
+				</ul>
+			</div>
+			<div class='o-subs-card__read-more'>Read more</div>
+		</div>
+	</div>
+	<div class="o-subs-card o-subs-card--b2b" data-o-component="o-subs-card">
+		<div class="o-subs-card__top">
+			<div class="o-subs-card__copy-title">Get the FT for your team or business</div>
```

Comment:
Missing the `o-subs-card__copy` wrapper on the b2b variant?

Reply:
Like we said, this was left over on discounts, so it's been deleted.